Bottom: aa46dbea57ca238eb17c9d6d90eeeb934e27a0b9
Top:    541cd8d79e0e2abd63ff7560497a720dc218206a
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-17 12:46:04 -0700

Refresh of more-reset

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
index 93414ea9874..e9f31212946 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
@@ -278,9 +278,7 @@ PythonByteArray::PythonByteArray(llvm::ArrayRef<uint8_t> bytes)
 
 PythonByteArray::PythonByteArray(const uint8_t *bytes, size_t length) {
   const char *str = reinterpret_cast<const char *>(bytes);
-  PyObject *obj = PyByteArray_FromStringAndSize(str, length);
-  assert(obj);
-  *this = Take<PythonByteArray>(obj);
+  *this = Take<PythonByteArray>(PyByteArray_FromStringAndSize(str, length));
 }
 
 bool PythonByteArray::Check(PyObject *py_obj) {
@@ -794,9 +792,7 @@ bool PythonModule::Check(PyObject *py_obj) {
 PythonDictionary PythonModule::GetDictionary() const {
   if (!IsValid())
     return PythonDictionary();
-  PyObject *obj = PyModule_GetDict(m_py_obj);
-  assert(obj);
-  return Retain<PythonDictionary>(obj);
+  return Retain<PythonDictionary>(PyModule_GetDict(m_py_obj));
 }
 
 bool PythonCallable::Check(PyObject *py_obj) {
